src/Microsoft.Health.Fhir.Core/Features/Context/IFhirRequestContext.cs

```diff
@@ -29,5 +29,10 @@ public interface IFhirRequestContext : IRequestContext
         /// Indicates whether this running as part of a background task instead of an HTTP request
         /// </summary>
         bool IsBackgroundTask { get; set; }
+
+        /// <summary>
+        /// A weakly-typed property bag
```

Comment:
Sounds similar to https://docs.microsoft.com/en-us/dotnet/api/system.web.httpcontext.items?view=netframework-4.8. Perhaps expand the comment the intended use?